apps/dashboard/app/views/batch_connect/sessions/connections/_sessions.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>
+	<strong> <%= t('dashboard.batch_connect_sessions_stats_host') %> </strong>
+	<%= " " %>
```

Comment:
If we want a space here just have a space in the HTML, switching back into a Ruby context is unnecessary.